src/resources/addresses/AddressList.tsx

```diff
@@ -17,7 +17,7 @@ export default function AddressList(): React.ReactElement {
       <Typography variant='h5' fontWeight={'bold'} style={{ padding: ' 15px' }}>
         Addresses
       </Typography>
-      <Datagrid rowClick='show' bulkActionButtons={<></>}>
+      <Datagrid rowClick='show' bulkActionButtons={false}>
         <TextField<Address> source='fullAddress' />
```

Comment:
Hello @TahaKhanAbdalli - I did some googling, and found that the above removes the ability to select items, since that's no longer relevant.